iab/draft-iab-protocol-transitions.xml

```diff
@@ -913,68 +915,133 @@ Suzanne Woolf
+          included:</t>
+
+          <t><list style="symbols">
+            <t>Pipelining - the ability to have more than one request to a server on the wire, to
+            mitigate Head of Line Blocking issues.</t>
+
+            <t>Expect/Continue - the ability to request that a server indicate that a request
+            header set is serviceable, so that the client can continue to send a request body with
+            reduced risk of it being rejected (thereby sacrificing the connection).</t>
+          </list></t>
+
+          <t>HTTP/2 <xref target="RFC7540"/> was largely aimed at improving performance. The
+          primary improvement was the introduction of request multiplexing, which is supported by
+          request prioritization and flow control. It also introduced header compression <xref
+          target="RFC7541"/> and binary framing; this made it completely backwards incompatible on
+          the wire, but still semantically compatible with previous versions of the protocol.</t>
+
+          <section title="Protocol Versioning, Extensions and 'Grease'">
+
+            <t>During the development of HTTP/1.1, there was a fair amount of confusing regarding
+            the semantics of HTTP version numbers, resulting in <xref target="RFC2145"/>. Later, it
+            was felt that minor versioning in a protocol caused more confusion than it was worth,
+            and so HTTP/2.0 became HTTP/2.</t>
+
+            <t>This decision was informed by the observation that many implementations ignored the
+            major version number of the protocol, or misinterpreted it. As is the case with many
+            protocol extension points, HTTP versioning had failed to be "greased" by change often
+            enough, and so had become "rusted" so that only a limited range of values could
+            interoperate.</t>
+
+            <t>This phenomenon has been observed in other protocols, such as TLS, and there are
+            active efforts to identify extension points that are in need of such "grease" so that
+            they remain flexible. This can be achieved both by good extension point design, and
+            through use of unplanned values by common implementations.</t>
+
+            <t>Besides the protocol version, HTTP's extension points that are well-greased include
+            header fields, status codes, media types and cache-control extensions; HTTP methods,
+            content-encodings and chunk-extensions enjoy less flexibility, and need to be extended
+            more cautiously.</t>
+
+          </section>
+          <section title="Limits on Changes in Major Versions">
+
+            <t>Although a new "major" version is an opportunity to break compatibility with the old
+            one and improve multiple aspects of a protocol, care needs to be taken to assure that
+            it can be successfully deployed.</t>
```

Comment:
I would try to concentrate on HTTP-specific statements here.  Instead of the first two paragraphs:

> Each update to the "major" version of HTTP have been accompanied by changes that weren't compatible with previous versions.  This was not uniformly successful given the diversity and scale of deployment and implementations.